lab-claudia/ng-template/app/view/landing/_landing.scss

```diff
@@ -0,0 +1,53 @@
+@import  "theme";
+
+.landing {
+  .join-inner-container {
+    padding: 5%;
+    margin-top: 10%;
+    margin-left: auto;
+    margin-right: auto;
+    width: 100%;
+    background-color: $app-secondary;
+    transition: 1s;
+
+    button {
+      color: $app-secondary;
+    }
+
+    input {
+      margin-top: 2%;
+
+      &[type='submit'] {
+        color: $app-secondary;
+      }
+    }
+
+    p {
+      padding: 2%;
+      display: inline-block;
+      color: $app-white;
+    }
+
+    a {
+      display: inline-block;
+      font-size: 1.2em;
+      text-decoration: none;
+    }
+
+    @media (min-width: 400px) {
+      width: 90%;
+    }
+
+    @media (min-width: 600px) {
+      width: 70%;
+    }
+
+    @media (min-width: 1000px) {
+      width: 50%;
+    }
+
+    @media (min-width: 1200px) {
```

Comment:
@loomnugget yay!  media queries look good _but_ try using device specifics for more control - you shouldnt style for a specific device _but_ i use IOS device widths as a baseline:

https://css-tricks.com/snippets/css/media-queries-for-standard-devices/